scalapb-runtime/src/main/scala/scalapb/GeneratedMessageCompanion.scala

```diff
@@ -67,7 +67,7 @@ trait GeneratedOneof extends Any with Product with Serializable {
   def valueOption: Option[ValueType] = if (isDefined) Some(value) else None
 }
 
-trait GeneratedOneofCompanion
+trait GeneratedOneofCompanion extends Serializable
```

Comment:
this doe snot seem to be used anymore, should it be deprecated?

Reply:
Nice catch. Looks like it has never been used. Let's mark it as deprecated.